robot.py

```diff
@@ -258,6 +258,11 @@ def teleopPeriodic(self) -> None:
         ):
             self.reef_intake.intake()
 
+        if self.gamepad.getLeftBumperButton() and self.reef_intake.is_executing:
+            self.reef_intake.is_aligning = True
+        else:
+            self.reef_intake.is_aligning = False

```

Comment:
Not that it matters now but reaching into the class like this is generally a bit of a smell in our code. you are better off creating `.align()` interface in the reef intake class that performs this operation and otherwise it resets to false so it doesn't toggle like this. cleaner interface and fewer implementation details leak out to `robot.py`.